web_poet/fields.py

```diff
@@ -216,3 +225,86 @@ def _without_unsupported_field_names(
     if item_field_names is None:  # item_cls doesn't define field names upfront
         return field_names[:]
     return list(set(field_names) & set(item_field_names))
+
+
+@attrs.define
+class SelectFields:
+    """This is used as a dependency in :class:`~.ItemPage` to control which
+    fields to populate its returned item class.
```

Comment:
I wonder if we should rename it, and maybe even move (though not sure), because it's not necessarily about web-poet fields used to populate the item class. It's about the item attributes user wants to get in the output. Page object may not use fields at all.

Reply:
Brainstorm: `SelectAttributes`, `SelectAttrs`, `SelectKeys`, `AttributeFilter`, `AttrFilter`, `KeyFilter`, `ItemFilter`, `FilterAttributes`, `FilterAttrs`, `FilterKeys`, `FilterItem`.

Reply:
I'm inclined to have the word _"item"_ in there for clarity. `ItemFilter` could mean that the page object supports different types of items and a filter is needed to choose the right one. Somehow, we may want to also include the word _"attributes"_ for better clarity.

How about `ItemAttributes`?